Dockerfile

```diff
@@ -2,4 +2,5 @@ FROM python:2.7
 WORKDIR /code
 ADD . /code
 RUN python setup.py develop
+RUN pip install cliquet[postgresql]
```

Comment:
I made it optionnal as well in https://github.com/mozilla-services/kinto/pull/100